blamo/lib/Test/test.dart

```diff
@@ -150,21 +150,27 @@ class _TestPageState extends State<TestPage> {
                               ),
                               FormBuilderTextField(
                                 textInputAction: TextInputAction.next,
+                                keyboardType: TextInputType.number,
                                 focusNode: formNodes[1],
                                 attribute: 'beginTestDepth',
-                                validators: [FormBuilderValidators.numeric()],
+                                validators: [FormBuilderValidators.numeric(), FormBuilderValidators.max(0)],
                                 decoration: InputDecoration(labelText: "Begin Test Depth (-m)"),
```

Comment:
Should we remove the negative from the unit here so it'd read "(m)?" Currently (-m) could imply that whatever the user inputs will be made negative after the fact, which isn't the case.